build.gradle

```diff
@@ -115,12 +114,12 @@ dependencies {
 
     implementation 'org.springframework.boot:spring-boot-starter-oauth2-resource-server'
 
-    implementation 'com.nimbusds:nimbus-jose-jwt:8.16' // pinned due to CVE-2019-17195 in 7.8.1 brought in by spring-boot-starter-oauth2-resource-server-2.2.5.RELEASE
+    implementation 'com.nimbusds:nimbus-jose-jwt:8.17'
 
     implementation 'org.hibernate:hibernate-java8:5.4.10.Final'
```

Comment:
can be removed now too - pinning to an old hibernate artifact.  Should be changed to `hibernate-core`, which can then be bumped - or even better removed as included in data-jpa

Reply:
removed